docs/home/howto/LOGGING.md

```diff
@@ -181,64 +186,66 @@ There is example of policy below. Be aware that this is only example, and it nee
 }
 ```
 
-Example above shows configuration with rollover daily or when index achieve 1 GB size. Indexes older than 14 days will
+Example above shows configuration with rollover index policy on a daily basis or when the index achieve 1 GB size. Indexes older than 14 days will
 be deleted. States and conditionals could be combined. Please
-see [policies](https://opendistro.github.io/for-elasticsearch-docs/docs/im/ism/policies/) documentation for more
+see [policies](https://opensearch.org/docs/latest/im-plugin/ism/policies//) documentation for more
 details.
 
-`Apply Policy`
+<br>
+
+#### Apply Policy
 
-To apply policy use similar API request as presented below:
+To apply a policy you can use similar API request as presented below:
 
 ```
-PUT _template/template_01
+PUT _index_template/ism_rollover
 ```
 
 ```json
 {
   "index_patterns": ["filebeat*"],
   "settings": {
-    "opendistro.index_state_management.rollover_alias": "filebeat"
-    "opendistro.index_state_management.policy_id": "epi_policy"
+    "plugins.index_state_management.rollover_alias": "filebeat"
+    "plugins.index_state_management.policy_id": "epi_policy"
   }
 }
 ```
 
 After applying this policy, every new index created under this one will apply to it. There is also possibility to apply
-policy to already existing policies by assigning them to policy in Index Management Kibana panel.
+policy to already existing policies by assigning them to policy in dashboard Index Management panel.
 
-## How to export Kibana reports to CSV format
+## How to export Dashboards reports
 
-Since v1.0 Epiphany provides the possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for
-Elasticsearch Kibana reports feature.
+Since v1.0 Epiphany provides the possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature. And after migrating from Elastic stack to OpenSearch stack you can make use of the OpenSearch Reporting feature a choieve this and more. 
```

Comment:
```suggestion
Since v1.0 Epiphany provides the possibility to export reports from Kibana to CSV, PNG or PDF using the Open Distro for Elasticsearch Kibana reports feature. And after migrating from Elastic stack to OpenSearch stack you can make use of the OpenSearch Reporting feature a achieve this and more. 
```